tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -292,14 +291,68 @@ async def test_get_dbcheckpoint_info_request_cmd(self, bft_network, tracker):
         for replica_id in range(len(bft_network.all_replicas())):
             self.verify_snapshot_is_available(bft_network, replica_id, last_blockId)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    @verify_linearizability()
+    async def test_get_dbcheckpoint_info_request_cmd_with_bft(self, bft_network, tracker):
+        """
+            sends a getdbCheckpointInfoRequest command to test for n-f replicas for created dbcheckpoints.
+        """
+        bft_network.start_all_replicas()
+        client = bft_network.random_client()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network, tracker)
+        for i in range(300):
+            await skvbc.send_write_kv_set()
+        # There will be 2 dbcheckpoints created on stable seq num 150 and 300.
+        await self.wait_for_stable_checkpoint(bft_network, bft_network.all_replicas(), 300)
+        await self.wait_for_created_snapshots_metric(bft_network, 2)
+
+        op = operator.Operator(bft_network.config, client, bft_network.builddir)
+        rep = await op.get_dbcheckpoint_info_request(False)
+        data = cmf_msgs.ReconfigurationResponse.deserialize(rep)[0]
+        self.assertTrue(data.success)
+        for r in client.get_rsi_replies().values():
+            res = cmf_msgs.ReconfigurationResponse.deserialize(r)
+            self.assertEqual(len(res[0].response.db_checkpoint_info), 2)
+            dbcheckpoint_info_list = res[0].response.db_checkpoint_info
+            self.assertTrue(any(dbcheckpoint_info.seq_num == 150 for dbcheckpoint_info in dbcheckpoint_info_list))
+            self.assertTrue(any(dbcheckpoint_info.seq_num == 300 for dbcheckpoint_info in dbcheckpoint_info_list))
+        last_blockId =  await bft_network.get_metric(0, bft_network, "Gauges", "lastDbCheckpointBlockId", component="rocksdbCheckpoint")
```

Comment:
we can verify metrics on all replicas. Please see https://github.com/vmware/concord-bft/pull/2228

Reply:
Fixed for this TC. for rest it is already fixed under #2228